src/coreclr/jit/emitarm64.cpp

```diff
@@ -16128,4 +16209,181 @@ bool emitter::IsRedundantLdStr(
 
     return false;
 }
+
+//-----------------------------------------------------------------------------------
+// ReplacedLdrStr: Potentially, overwrite a previously-emitted
+//                 "ldr" or "str" instruction with an "ldp" or
+//                 "stp" instruction.
+//
+// Arguments:
+//     ins      - The instruction code
+//     reg1Attr - The emit attribute for register 1
+//     reg1     - Register 1 number
+//     reg2     - Register 2 number
+//     imm      - Immediate offset, prior to scaling by operand size
+//     size     - Operand size
+//     fmt      - Instruction format
+//
+// Return Value:
+//    "true" if the previous instruction HAS been overwritten.
+
+bool emitter::ReplacedLdrStr(
+    instruction ins, emitAttr reg1Attr, regNumber reg1, regNumber reg2, ssize_t imm, emitAttr size, insFormat fmt)
+{
+    if (emitComp->opts.OptimizationEnabled())
```

Comment:
nit:

```c++
if (!emitComp->opts.OptimizationEnabled())
{
  return false;
}
```

Reply:
I have made this change.

Reply:
I don't see it here. Could you please push them?